app/util/mapIconUtils.js

```diff
@@ -70,6 +71,15 @@ export function drawRoundIcon(tile, geom, type, large) {
       );
 
       tile.ctx.fill();
+
+      if (platformNumber && hubRadius > 7) {
```

Comment:
It is dependent on if it fits inside the hub, which requires exactly 14 pixels. The size of the hub varies e.g. if it is selected and therefore is not bound to any specific zoom level.